### DISCUSSION TITLE
失败的任务一定会被禁用吗?

### DISCUSSION THREAD

**Discussion by xyzdev-cell** [qd-today/qd#441] [Q&A] [answered]
有时候网站只是不稳定, 连不上, 希望失败就记录个失败

现在一失败就禁用, 我还需要重新去启用

**Comment by a76yyyy** [comment, marked as answer]
在任务新建和修改界面设置重试次数

![重试次数](https://github.com/qd-today/qd/assets/56478790/767def1f-116e-40a8-a3a0-93c6f0228751)

**Comment by xyzdev-cell** [comment]
嗯, 误解了, 因为默认重试是立即生效的
重试间隔设置成每天重试比较符合我的需求了
